tests/MongoDB.Driver.Tests/OcspIntegrationTests.cs

```diff
@@ -0,0 +1,144 @@
+/* Copyright 2020–present MongoDB Inc.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+using System;
+using FluentAssertions;
+using MongoDB.Bson;
+using MongoDB.Driver.Core;
+using MongoDB.Driver.Core.Configuration;
+using MongoDB.Driver.Core.TestHelpers.XunitExtensions;
+using MongoDB.Driver.TestHelpers;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace MongoDB.Driver.Tests
+{
+    public class OcspIntegrationTests
+    {
+        private static readonly string _ocspOutcomeEnvironmentVariableName = "OCSP_TLS_SHOULD_SUCCEED";
+        private readonly ITestOutputHelper _output;
```

Comment:
we don't use it :)

Reply:
We don't, but it's helpful when testing locally :)

Reply:
yep, but I think it's better to keep only code that can be executed. Possible cases probably it's better to add manually if it's required. At least I didn't see code which is left only to use it in local debugging :)

Reply:
I agree that unused code would be inappropriate for the driver projects. For test projects, I honestly don't mind having conveniently stubbed out test functions, but given the pending beta release, I'm fine with removing this.